pyproject.toml

```diff
@@ -1,49 +1,51 @@
+[tool.pdm]
+
+[project]
+name = "highspy"
+version = "1.5.3"
+description = "A thin set of pybind11 wrappers to HiGHS"
+authors = [
+    {name = "HiGHS developers", email = "highsopt@gmail.com"},
+]
+dependencies = []
+requires-python = ">=3.8"
+readme = "README.md"
+license = {text = "MIT"}
+
+[project.urls]
+"Source Code" = "https://github.com/ERGO-Code/HiGHS"
+"Bug Tracker" = "https://github.com/ERGO-Code/HiGHS/issues"
+
 [build-system]
-# Minimum requirements for the build system to execute.
-requires = [
-  "setuptools>=45",
-  "pybind11>=2.4",
-  "pyomo>=6.0",
-  "wheel>=0.2",
-] 
+requires = ["pybind11", "meson-python"]
```

Comment:
Due to the use of forcefallback, the pybind11 requirement here is installed but not used -- meson doesn't bother checking for the pybind11-config script installed by the pybind11 wheel into the `pip` / `python -m build` isolated build venv. It goes straight to initializing pybind11 as a meson subproject and retrieves the pybind11 `declare_dependency()` object.

Reply:
Ah neat, I didn't realize that. Updated :)